docs/content/en/docs/install/install.md

```diff
@@ -5,15 +5,9 @@ weight: 35
 hidechildren: false # this flag hides all sub-pages in the sidebar-multicard.html
 ---
 
-Keptn must be installed, enabled, and integrated
+Keptn must be installed and integrated
 into each Kubernetes cluster you want to monitor.
-This is because Keptn communicates with the Kubernetes scheduler
-for tasks such as enforcing checks natively,
-stopping a deployment from proceeding when criteria are not met,
-doing post-deployment evaluations
-and tracing all activities of all deployment [workloads](https://kubernetes.io/docs/concepts/workloads/) on the cluster.
-
-Keptn v.0.9.0 and later is installed using
+Keptn v.0.9.0 and later is packaged as
```

Comment:
I would stick to installed, packaged is not the right word here

Reply:
Thanks!  I reverted it to "installed using"